plugins/de.cau.cs.kieler.klighd.standalone/src/de/cau/cs/kieler/klighd/standalone/KlighdStandaloneSetup.java

```diff
@@ -129,6 +131,11 @@ protected void registerFonts() {
         final GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
         for (int i = 0; i < filePaths.length; ++i) {
             try (InputStream fontStream = this.getClass().getResourceAsStream(filePaths[i])) {
+                
+                // TODO Hacky workaround for opening fonts
+                String path = this.getClass().getProtectionDomain().getCodeSource().getLocation().getPath() + ".." + filePaths[i];
+                InputStream temp = new FileInputStream(new File(path));
+                
```

Comment:
What is this even supposed to do? Do the fonts not work in some case where they should be loaded?